### ISSUE TITLE
WFL editor crashes if I click on some Tasks

### ISSUE THREAD

**Issue by wingiti** [StackStorm/st2#5185]
## SUMMARY

The new WFL editor crashes if I click on some specific tasks. The affected actions are all part of self developed packs. It affects multiple actions belonging to multiple packs.
The same action is used two times in the workflow, one task can be edited the other one crashes the editor. -> I don't think it is related to my pack and action. But till now I have not identified what exactly makes the difference for the tasks working and the ones not working.

### STACKSTORM VERSION

st2 3.4.0, on Python 3.6.8

##### OS, environment, install method

redhat:enterprise_linux:7.9

## Steps to reproduce the problem

I can not share my current workflow and self developed packs, but I will check later if I can create an example workflow. Therefore I need to know what exactly the issue is, maybe someone can already give me a hint by looking at the error I get from Firefox or can tell me what I should check in advance. Otherwise I need more time to identify it on my own.

## Expected Results

Move and edit tasks like others.

## Actual Results

The editor crashes. (white page) used Browser: Firefox 86.0

Debug Output:
<img width="1664" alt="Bildschirmfoto 2021-03-10 um 11 15 50" src="https://user-images.githubusercontent.com/73697233/110613967-23c12180-8192-11eb-883a-8762a952de19.png">

**Comment by amanda11** [comment]
I know the workflow composer didn't handle delay or retry parameters.
So just wondering if that might explain it?

**Comment by amanda11** [comment]
Possible duplication of: https://github.com/StackStorm/st2web/issues/855. In that case all the failing tasks, had an input parameter of array, and a variable being passed to them. Could this be same problem @wingiti ?

**Comment by wingiti** [comment]
Yes I think it is the same kind of issue and can confirm the behaviour regarding the side panel. I also recognized now with the hint on having a look on the input parameter, that it is not only about actions with array input. It also occurs if I use `ctx().get(), ctx().switch, ctx().replace() or ctx().substring()` for normal string input parameters.
